ferrocene/doc/internal-procedures/src/upstream-pulls.rst

```diff
@@ -38,10 +38,21 @@ Start by checking out the PR. The name of the branch is
 Fix conflict
 ^^^^^^^^^^^^
 
-To find the conflicting files run following command::
+To find the conflicting files locally run the following command::
 
   ferrocene/ci/scripts/detect-conflict-markers.py
 
+The failures are also visible on CI, which is useful to track at what 
+stage the failure happened and how it is affecting the build.
+
+.. figure:: figures/ci-failure.png
```

Comment:
These screenshots do not feel related to merge conflictsx, should those errors not occur in the normal commit check job? https://github.com/ferrocene/ferrocene/blob/30d0f4f9ffc4506cbc6ba051d1c886b2b8181012/.github/workflows/commit.yml#L38-L39

Reply:
was this handled... I see the screenshot is still there